### PR TITLE
Black!

### PR DESCRIPTION
**Pull request by reinout**
I ran this command:

    $ black .

Note that I added a `pyproject.toml` to configure black to exclude the `external/` directory.

### CODE REVIEW

**Comment by reinout** [comment]
Some extra notes: I trust black now, so I removed the line length warning in pycodestyle/pep8.
Any lines that are too long are too long for a reason :-)

File: __init__.py [hunk 1]
```diff
@@ -32,34 +32,36 @@
...
-    log('Use local installation of pyqtgraph ')
+
+    log("Use local installation of pyqtgraph ")
 except Exception:
     # TODO: fix this error (which is the reason of this exception):
     # Exception: PyQtGraph requires either PyQt4 or PySide; neither package
     # could be imported.
-    msg = "Error: Exception while loading pyqtgraph. Probably couldn't "\
-          "import PyQt"
+    msg = "Error: Exception while loading pyqtgraph. Probably couldn't " "import PyQt"
```
Comment (RichardB9) [inline __init__.py]: String split over multiple lines are concatenated weirdly

File: __init__.py [hunk 2]
```diff
@@ -32,34 +32,36 @@
...
     pop_up_info(msg)
 
 try:
     import lizard_connector  # noqa
 except ImportError as e:
-    pop_up_info("Error loading lizard_connector from "
-                "'external' subdirectory. error %s" % e)
+    pop_up_info(
+        "Error loading lizard_connector from " "'external' subdirectory. error %s" % e
+    )
```
Comment (RichardB9) [inline __init__.py:64]: See above

File: __init__.py [hunk 3]
```diff
@@ -106,30 +116,39 @@
...
     # Note: threedigrid depends on h5py and netCDF, so don't import it directly
     # (see above).
-    imp.find_module('threedigrid')  # noqa
+    imp.find_module("threedigrid")  # noqa
 except ImportError as e:
-    pop_up_info("Error loading threedigrid from "
-                "'external' subdirectory. error %s" % e)
+    pop_up_info(
+        "Error loading threedigrid from " "'external' subdirectory. error %s" % e
+    )
```
Comment (RichardB9) [inline __init__.py]: See above

File: commands/Tools/Step 3 - Modify schematisation/add_connected_points.py [hunk 4]
```diff
@@ -4,16 +4,13 @@
...
-    QgsGeometry
-)
+from qgis.core import QgsProject, QgsFeatureRequest, QgsFeature, QgsGeometry
 
 from ThreeDiToolbox.utils.user_messages import messagebar_message
 from ThreeDiToolbox.utils import constants
-from ThreeDiToolbox.views.modify_schematisation_dialogs import AddCoonnectedPointsDialogWidget  # noqa
+from ThreeDiToolbox.views.modify_schematisation_dialogs import (
+    AddCoonnectedPointsDialogWidget,
+)  # noqa
```
Comment (RichardB9) [inline commands/Tools/Step 3 - Modify schematisation/add_connected_points.py]: `#noqa` not needed anymore with new formatting

File: commands/Tools/Step 3 - Modify schematisation/create_breach_locations.py [hunk 5]
```diff
@@ -7,7 +7,9 @@
 
 from ThreeDiToolbox.utils.user_messages import messagebar_message
 from ThreeDiToolbox.utils.user_messages import progress_bar
-from ThreeDiToolbox.views.modify_schematisation_dialogs import CreateBreachLocationsDialogWidget  # noqa
+from ThreeDiToolbox.views.modify_schematisation_dialogs import (
+    CreateBreachLocationsDialogWidget,
+)  # noqa
```
Comment (RichardB9) [inline commands/Tools/Step 3 - Modify schematisation/create_breach_locations.py]: noqa not needed

File: commands/Tools/Step 3 - Modify schematisation/predict_calc_points.py [hunk 6]
```diff
@@ -8,7 +8,9 @@
 from ThreeDiToolbox.utils import constants
 from ThreeDiToolbox.utils.user_messages import messagebar_message
 from ThreeDiToolbox.utils.user_messages import pop_up_question
-from ThreeDiToolbox.views.modify_schematisation_dialogs import PredictCalcPointsDialogWidget  # noqa
+from ThreeDiToolbox.views.modify_schematisation_dialogs import (
+    PredictCalcPointsDialogWidget,
+)  # noqa
```
Comment (RichardB9) [inline commands/Tools/Step 3 - Modify schematisation/predict_calc_points.py]: noqa not needed

File: misc_tools.py [hunk 7]
```diff
@@ -117,8 +112,9 @@ def run(self):
                     os.remove(f)
                 except OSError:
                     pop_up_info("Failed to delete %s." % f)
-            pop_up_info("Cache cleared. You may need to restart QGIS and "
-                        "reload your data.")
+            pop_up_info(
+                "Cache cleared. You may need to restart QGIS and " "reload your data."
+            )
```
Comment (RichardB9) [inline misc_tools.py:118]: Weird string concat

File: test/test_raster_checker.py [hunk 8]
```diff
@@ -2,8 +2,14 @@
...
-from ThreeDiToolbox.utils.raster_checker_prework import (DataModelSource, RasterCheckerEntrees)  # noqa
-from ThreeDiToolbox.utils.raster_checker_log import (RasterCheckerResults, RasterCheckerProgressBar)  # noqa
+from ThreeDiToolbox.utils.raster_checker_prework import (
+    DataModelSource,
+    RasterCheckerEntrees,
+)  # noqa
+from ThreeDiToolbox.utils.raster_checker_log import (
+    RasterCheckerResults,
+    RasterCheckerProgressBar,
+)  # noqa
```
Comment (RichardB9) [inline test/test_raster_checker.py]: noqa not needed anymore

**Review by RichardB9 — APPROVED** [review]
Nice tool! Everything is more consistent!

Some `noqa` are not needed anymore.
A few strings split over multiple lines are concatenated weirdly.

I probably missed some due to the size of the changes

**Comment by reinout** [comment]
Good catch!
I've zapped quite a lot of the `# noqa` and I've searched the code for `" "` instances that could be removed.

**Pull request merged**